tasks/main.yml

```diff
@@ -43,7 +43,7 @@
 - name: Configure /vsmagent/agent_hostname
   tlconfig:
     param: /vsmagent/agent_hostname
-    value: "{{ thinlinc_agent_hostname or ansible_fqdn }}"
+    value: "{{ thinlinc_agent_hostname or ansible_hostname }}"
```

Comment:
`ansible_hostname` is not always an externally valid name, so I think we're just trading one issue for another here.

What scenario are you trying to fix here?

Reply:
Maybe this was a bit overdone, ansible_fqdn was returning wrong values so i changed it to ansible_hostname.